modules/Core/pages/oauth.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+const PAGE = 'oauth';
+require_once(ROOT_PATH . '/core/templates/frontend_init.php');
```

Comment:
Why do we need frontend_init here?

Reply:
So the template variable is properly assigned when the 2FA page is rendered